PowerShell/ScubaGear/Modules/Providers/ProviderHelpers/CommandTracker.psm1

```diff
@@ -1,5 +1,5 @@
 Import-Module -Name $PSScriptRoot/../ExportEXOProvider.psm1 -Function Get-ScubaSpfRecord, Get-ScubaDkimRecord, Get-ScubaDmarcRecord
-Import-Module -Name $PSScriptRoot/../ExportAADProvider.psm1 -Function Get-PrivilegedRole, Get-PrivilegedUser
+Import-Module -Name $PSScriptRoot/../ExportAADProvider.psm1 -Function Get-PrivilegedRole, Get-PrivilegedUser, Get-PrivilegedServicePrincipal
```

Comment:
I don't see `Get-PrivilegedServicePrincipal` as a function in your branch's [ExportAADProvider.psm1](https://github.com/cisagov/ScubaGear/blob/934_service_principals_to_aad_report/PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1).